packages/cmd/__tests__/CommandProcessor.test.ts

```diff
@@ -1600,46 +1608,6 @@ describe("Command Processor", () => {
         expect(commandResponse).toMatchSnapshot();
     });
 
-    it("should invoke the handler and return success=true if the handler was successful", async () => {
```

Comment:
Was this test supposed to be removed?

Reply:
I think so, it appeared to be a duplicate of this test which still exists in the "use-eslint" branch: https://github.com/zowe/imperative/blob/e8b7aee9a77e2f2075423c317096feb34a011483/packages/cmd/__tests__/CommandProcessor.test.ts#L1327-L1365